### ISSUE TITLE
Template level event binding doesn't work with meteor-jade?

### ISSUE THREAD

**Issue by neoromantic** [peerlibrary/meteor-blaze-components#113]
I'm using mquandalle:jade package to have '.jade' templates in my projects.

When I'm trying to bind events within template, as you recommend to do, it fails. 

For example:

component.jade:

```
template(name="MyComponent")
  button(onClick=clickHappened)
```

component.coffee

```
class MyComponent extends BlazeComponent
   clickHappened: -> 1
```

When component renders, I get an error: 

`Error: Event handler not a function: 1`

What happens, as far as I understand is that jade compiler executes 'clickHappened' method instead of passing it as an argument.  

Is there any way (syntax?) to use Jade with BlazeComponents?

**Comment by mitar** [comment]
Yes, this is something @mquandalle should add to his compiler. [Here is code](https://github.com/peerlibrary/meteor-blaze-components/blob/master/patch-compiling.js) for HTML files.

**Comment by ciclopes** [comment]
As far as it needs a function, have you tried `clickHappened: -> -> 1`?
It may be a little bit ugly workaround, but it should work.

**Comment by comerc** [comment]
Hmm, [blaze-magic-events](https://github.com/themeteorites/blaze-magic-events/issues/3) works fine.

**Comment by mitar** [comment]
Because magic events seems to renders event handlers to DOM to define events. This is much uglier approach.
